.gitlab-ci-check-golang-static.yml

```diff
@@ -40,7 +40,7 @@ test:static:
     # Test if code was formatted with 'go fmt'
     # Command will format code and return modified files
     # fail if any have been modified.
-    - if [ -n "$(go fmt ./...)" ]; then echo 'Code is not formatted with "go fmt"'; false; fi
+    - if [ -n "$(go fmt ./... | grep -v ^$)" ]; then echo 'Code is not formatted with "go fmt"'; false; fi
```

Comment:
I think you need to single-quote `^$`. I'm not sure when the `$` is at the end like that, but I think it's a safer approach anyway.